tutor/src/components/new-course/select-dates.cjsx

```diff
@@ -4,9 +4,10 @@ BS = require 'react-bootstrap'
 classnames = require 'classnames'
 partial = require 'lodash/partial'
 isEqual = require 'lodash/isEqual'
-S = require '../../helpers/string'
+
 {NewCourseActions, NewCourseStore} = require '../../flux/new-course'
 {OfferingsStore} = require '../../flux/offerings'
+{CourseChoiceItem} = require './choice'
 
 QUARTERS =
   '2017Q4': 'Fall 2017'
```

Comment:
Hm, I just noticed that I left the old stub data`QUARTERS` in the file.  If we do further updates that can be removed.